data/modules/scripts/gamestore/init.lua

```diff
@@ -323,7 +323,8 @@ function parseRequestStoreOffers(playerId, msg)
 		local category = nil
 
 		if subAction == 0 then
-			category = GameStore.getCategoryByName("Premium Time")
+			local categoryName = ((configManager.getBoolean(configKeys.VIP_SYSTEM_ENABLED) and "Vip Shop") or "Premium Time")
```

Comment:
local categoryName = (configManager.getBoolean(configKeys.VIP_SYSTEM_ENABLED) and "Vip Shop") or "Premium Time")